scalableminds/webknossos
# Show dates in local timezone

Apparently, this has been the case before, anyway. However, this PR changes date formatting so that a tooltip is shown (on hover) which explains that the date is in local TZ. additionally, the date is formatted as utc in the tooltip.

![image](https://user-images.githubusercontent.com/2486553/42888385-74c82bc0-8aa8-11e8-9c3e-7cfc90ce93bc.png)


### URL of deployed dev instance (used for testing):
- https://localdaterendering.webknossos.xyz

### Steps to test:
- visit the corresponding pages of the change files and check that the dates are rendered properly

### Issues:
- fixes #2814

------
